docker/test

```diff
@@ -78,14 +82,21 @@ if [ "$EXPOSE_PORTS" == "1" ]; then
   SERVICE_PORTS_ARG="--service-ports"
 fi
 
-if [ "$DETACHED" == "1" ]; then
-    # Run detached, and don't remove the containers
-    docker-compose $PROJ_NAME $COMPOSE_ARGS run -d $SERVICE_PORTS_ARG candlepin $TEST_CMD
-    RETVAL=$?
+if [ "$INIT_CONTAINER" == "1" ]; then
+  docker-compose $PROJ_NAME $COMPOSE_ARGS up -d
```

Comment:
If it's not too much trouble, lets move this straight to podman-compose for cs9 for now, and we'll do cs8 in a separate PR

Reply:
done